ui/raidboss/data/triggers/twinning.js

```diff
@@ -0,0 +1,150 @@
+'use strict';
+
+// The Twinning
+
+[{
+  zoneRegex: /^The Twinning$/,
+  triggers: [
+    {
+      // The handling for these mechanics is similar enough it makes sense to combine the trigger
+      id: 'Twinning Impact + Pounce',
+      regex: / 1B:........:\y{Name}:....:....:(003[2-5]|005A)/,
+      suppressSeconds: 10,
+      alertText: {
+        en: 'Spread--Avoid cages',
+      },
+    },
+    {
+      id: 'Twinning Beastly Roar',
+      regex: / 14:3D64:Alpha Zaghnal starts using Beastly Roar/,
+      condition: function(data) {
+        return data.role == 'healer';
+      },
+      alertText: {
+        en: 'AoE Damage',
```

Comment:
I think most of these just say `aoe` for what it's worth.

Reply:
Noted for the revision.